src/boot/efi/meson.build

```diff
@@ -43,11 +43,7 @@ if not cc.has_header_symbol('efi.h', 'EFI_IMAGE_MACHINE_X64',
         subdir_done()
 endif
 
-objcopy = find_program('objcopy')
-efi_cc = get_option('efi-cc')
-if efi_cc.length() == 0
-        efi_cc = cc.cmd_array()
-endif
+objcopy = run_command(cc.cmd_array(), '-print-prog-name=objcopy', check: true).stdout().strip()
```

Comment:
`--print-prog-name`?? In the main meson.build, we use `--print-prog-name` for detecting `llvm-strip`.

Reply:
Both gcc and clang list this with one dash in `--help`, so I prefer that. They happily accept either, though.

Reply:
Indeed. Thanks.